packages/server/test/unit/browsers/electron_spec.js

```diff
@@ -470,6 +471,108 @@ describe('lib/browsers/electron', () => {
         })
       })
     })
+
+    describe('setUserAgent with experimentalModifyObstructiveThirdPartyCode', () => {
+      let userAgent
+
+      beforeEach(function () {
+        userAgent = ''
+        electron._getUserAgent.callsFake(() => userAgent)
```

Comment:
doing a `callsFake` over `returns` since `userAgent` is assigned dynamically in the test.